projects/vdk-plugins/vdk-oracle/src/vdk/plugin/oracle/ingest_to_oracle.py

```diff
@@ -225,16 +231,23 @@ def _populate_query_parameters_tuple(
 
         parameters = []
         for obj in payload:
-            row = tuple(
-                self._cast_to_correct_type(
-                    destination_table, column.lower(), obj.get(column.lower())
-                )
-                for column in columns
-            )
+            row = []
+            for column in columns:
```

Comment:
can you add a check that checks if the column has both lower and upper case vars, and if so to log.warn that this might lead to unexpected result or throw an exception that using both is currently not allowed